frontend/src/components/Navbar/Navbar.js

```diff
@@ -1,31 +1,45 @@
 import React, {Component} from 'react';
-import { MenuItems } from "./MenuItems";
 import { BrowserRouter as Router, Route, Switch, Link } from "react-router-dom";
 import './Navbar.css';
 
+export const MenuItems = [
```

Comment:
Is the export needed since MenuItems will only be accessed in this file?

Reply:
@ryanbuui @NirmalAgnihotri please take a look and address this!

Reply:
My bad, I forgot to remove that. It is not needed.